packages/kbn-search-index-documents/components/result/result_field.tsx

```diff
@@ -60,13 +65,84 @@ const iconMap: Record<string, string> = {
 };
 const defaultToken = 'questionInCircle';
 
+const PERMANENTLY_TRUNCATED_FIELDS = ['dense_vector', 'sparse_vector'];
+
+interface ResultValueProps {
+  fieldValue: string;
+  fieldType: string;
+  isExpanded?: boolean;
+}
+
+const ResultValue: React.FC<ResultValueProps> = ({ fieldValue, fieldType, isExpanded = false }) => {
```

Comment:
would like to see this component get it's own file. I know we break the 1 component per file rule all the time, but we should try not to.

Reply:
yeh i got lazy :) updated.